x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/ClassificationIT.java

```diff
@@ -444,4 +448,12 @@ private static void indexData(String sourceIndex, int numTrainingRows, int numNo
             assertThat(confusionMatrixResult.getOtherActualClassCount(), equalTo(0L));
         }
     }
+
+    private static void assertModelStatePersisted(String jobId) {
```

Comment:
Is it the same method as in `RegressionIT`? If so, it could be moved to `MlNativeDataFrameAnalyticsIntegTestCase`, just like `assertInferenceModelPersisted`.